### DISCUSSION TITLE
How can I test the v2.0 beta Function on Solana chain?

### DISCUSSION THREAD

**Discussion by huangli369** [WalletConnect/walletconnect-monorepo#1155] [Developer Support]
I have finished the v2.0 function, but I don't know how to test it on Solana chain.
Can you give me a test link to test? 
thank you.

**Comment by chadyj** [comment]
You can test with our v2 sample dapp here https://react-app.walletconnect.com/
Just select Solana.

**Reply by huangli369** [reply]
There are two different code.
One is "wc:1a1dd12561cb2e3413d21ec995df1c97ba416224689e082d9f652101a3a0a7a8@2?controller=false&publicKey=e727fc3bd7afa24d996f64298cc5817a720f51ff9341aa52aa5e3a7c6578284b&relay=%7B%22protocol%22%3A%22waku%22%7D".
Another is "wc:43865777b5a070f8a7e74f1a48321f61ae7d3384cd7c0a54dece27b3b6acf218@2?relay-protocol=waku&symKey=759fdc6929d037bd270637afee0a49ef66ceeee0e78d4bd835e701be266cb6cf".
iOS SDK can only pair the first code.
The link you provided generate the second code, cann't pair by iOS SDK. Error like this "Pairing URI string is invalid".
Can you tell me what I should do next.

**Reply by chadyj** [reply]
It works for me using the sample app.
Can you confirm you are using the latest beta 101?
https://github.com/WalletConnect/WalletConnectSwiftV2/releases/tag/v0.7.0-beta.101

**Reply by huangli369** [reply]
OK, I think I used the old version.
Do you have the dart language plugin?

**Reply by chadyj** [reply]
Great. Hope it works with the new version.
No dart for now.
